man/importexample.Rd

```diff
@@ -0,0 +1,16 @@
+\name{importexample}
+\docType{data}
+\alias{importexample}
+\title{Import Example}
+\description{
+	A minimum working example (MWE) to test the functions \code{\link{import.q.concourse}}, \code{\link{build.q.set}}, \code{\link{import.q.sorts}}, \code{\link{import.q.feedback}} and \code{\link{make.cards}}.
+  The example is too small to run an actual Q analysis.
+	To test out a real study with the same data structure, go to: \url{https://github.com/maxheld83/keyneson}.
+}
+\usage{importexample}
+\format{
+  Importexample is included as a directory in the package root, including subdirectories as documented in the package documentation, and on the package \href{https://github.com/aiorazabala/qmethod/wiki/Data-management}{wiki}.
```

Comment:
\code{Importexample} or "Importexample"

`...is included as a directory in the 'qmethod' package root folder [otherwise one can think of a pkg called 'root'], including subdirectories as documented in qmethod-package, and on the package...`